analyses/maven/matcher.go

```diff
@@ -0,0 +1,67 @@
+package maven
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/fabric8-analytics/cli-tools/analyses/driver"
+	"github.com/rs/zerolog/log"
+)
+
+var (
+	_ driver.StackAnalysisInterface = (*Matcher)(nil)
+)
+
+// Matcher is State Object for Maven
+type Matcher struct {
+	FilePath string
+}
+
+// Ecosystem implements driver.Matcher.
+func (*Matcher) Ecosystem() string { return "maven" }
+
+// DepsTreeFileName implements driver.Matcher.
+func (*Matcher) DepsTreeFileName() string { return "dependencies.txt" }
+
+// GeneratorDependencyTree creates dependencies.txt from pom.xml
+func (m *Matcher) GeneratorDependencyTree(manifestFilePath string) string {
+	log.Debug().Msgf("Executing: Generate dependencies.txt")
+	maven, err := exec.LookPath("mvn")
+	if err != nil {
+		log.Fatal().Err(err).Msgf("Please make sure Maven is installed. Hint: Check same by executing: mvn --version \n")
+	}
+	treePath, _ := filepath.Abs(filepath.Join(os.TempDir(), m.DepsTreeFileName()))
+	outcmd := fmt.Sprintf("-DoutputFile=%s", treePath)
+	cmd := exec.Command(maven, "--quiet", "clean", "-f", manifestFilePath)
+	cmd2 := exec.Command(maven, "--quiet", "org.apache.maven.plugins:maven-dependency-plugin:3.0.2:tree", "-f", manifestFilePath, outcmd, "-DoutputType=dot", "-DappendOutput=true")
```

Comment:
Instead of generic name like cmd, cm2, Use a right name to make the intent clear.
```suggestion
	cleanRepo := exec.Command(maven, "--quiet", "clean", "-f", manifestFilePath)
	dependencyTree := exec.Command(maven, "--quiet", "org.apache.maven.plugins:maven-dependency-plugin:3.0.2:tree", "-f", manifestFilePath, outcmd, "-DoutputType=dot", "-DappendOutput=true")
```

Reply:
Ack.